xbmc/network/upnp/UPnPServer.cpp

```diff
@@ -872,10 +873,10 @@ CUPnPServer::OnSearchContainer(PLT_ActionReference&          action,
                                const char*                   sort_criteria,
                                const PLT_HttpRequestContext& context)
 {
-  m_logger->debug("Received Search request for object '{}' with search '{}'", object_id,
-                  search_criteria);
+  m_logger->debug("Received Search request for encoded object '{}' (plain value: '{}') with search '{}'",
+                  object_id, DecodeObjectId(object_id).GetChars(), search_criteria);
```

Comment:
You're decoding `object_id` twice. Please use a local variable for the decoded `object_id`.